liberty-maven-plugin/src/it/tests/appsdirectory-set-appz-configured-it/src/main/resources/server.xml

```diff
@@ -0,0 +1,6 @@
+<server description="default server">
```

Comment:
Why do you have src/main/resources/server.xml and src/test/resources/server.xml?  I think you can delete the 2 in the src/main folders.